### DISCUSSION TITLE
Problem while setting it as an search engine

### DISCUSSION THREAD

**Discussion by ALMIGHTY-VC** [searxng/searxng#3862] [Q&A]
Please also stipulate if you are using a forked version of SearXNG and
include a link to the fork source code.
-->using fmhy version
**How did you install SearXNG?**
 i did not installed it.
**What happened?**
i tried to set it as a default search engine on the via browser app installed on my Android getting page not found for every search done through the browser toolbar i know the end is on my side please help me fix it

**How To Reproduce**
use via browser and set  https://searx.fmhy.net/ as the default search engine type anything and get the desired search results

**Expected behavior**
it should have saerched what i typed in the browser searchbar

**Screenshots & Logs**

![Screenshot_2024-09-28-23-59-28-117_mark via gp](https://github.com/user-attachments/assets/46f2bd62-62b0-4911-9106-96e4a309ef63)

**Additional context**

Tried setting up searxng. fmhy on via every search using the browser search bar gives page not found.

**Comment by ALMIGHTY-VC** [comment]
Welp sorry for not opening a discussuon

**Comment by return42** [comment]
> i tried to set it as a default search engine on the via browser app installed on my Android

Instance https://searx.fmhy.net/  did work in FFox (desktop) .. Android browsers often do not (well) support the [Autodiscovery of search plugins](https://developer.mozilla.org/en-US/docs/Web/OpenSearch#autodiscovery_of_search_plugins) based on the [OpenSearch protocol](https://github.com/dewitt/opensearch).  

Do https://en.wikipedia.org/ work in your Via-Browser app?

Related:

- https://github.com/searxng/searxng/issues/431

**Comment by ALMIGHTY-VC** [comment]
Well after setting searxng as search engine and putting the link you gave me on the browser searchbar it did landed me on the site👍🏻
